streams/src/test/java/org/apache/kafka/streams/integration/FanoutIntegrationTest.java

```diff
@@ -149,7 +145,9 @@ public String apply(String value) {
         consumerConfigB.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
         consumerConfigB.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class);
         consumerConfigB.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
-        List<String> actualValuesForB = IntegrationTestUtils.readValues(OUTPUT_TOPIC_B, consumerConfigB, inputValues.size());
+        List<String> actualValuesForB =
+            IntegrationTestUtils.waitUntilValuesMatch(consumerConfigB, OUTPUT_TOPIC_B, inputValues.size(),
+                inputValues.size(), IntegrationTestUtils.DEFAULT_TIMEOUT);
```

Comment:
Style nit: do we really want to break the line right after `=`? Since we can't avoid splitting the method invocation into two, it may make sense to just continue after `=` and end up with two lines instead of three.